WordPress/gutenberg
# CustomGradientPicker: use KeyboardEvent.code instead of KeyboardEvent.keyCode

## What?


Refactor the `CustomGradientPicker ` component to rely on `code` instead of `keyCode` for keyboard events

## Why?


[`keyCode`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) is deprecated, and replaced by [`code`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code)

## How?


Easy swap of values

## Testing Instructions





In Storybook, visit the gradient picker example, focus the gradient control points, and make sure that pressing arrow left/right actually moves the control points along the gradient bar.
